src/engine.js

```diff
@@ -9,62 +9,27 @@ const loadTasksFromLocalStorage = () => {
   }
 };
 
-// Save and update local storage
-const saveTasksToLocalStorage = () => {
-  localStorage.setItem('todoList', JSON.stringify(list));
-  const filteredList = list.filter((task) => !task.completed);
-  localStorage.setItem('todoList', JSON.stringify(filteredList));
-};
-
-// Display tasks
-const displayItemsByIndex = () => {
-  listContainer.innerHTML = '';
-
+// Event listeners for the checkboxes and delete buttons
+const attachEventListeners = () => {
   list.forEach((item) => {
-    const { description, completed, index } = item;
-
-    const listItem = document.createElement('div');
-    const descriptionSpan = document.createElement('span');
-    const completedRadio = document.createElement('input');
-    const edit = document.createElement('button');
-    const hr = document.createElement('hr');
-
-    edit.classList.add('edit-task');
-
-    listItem.classList.add('icon');
-
-    descriptionSpan.textContent = description;
-    completedRadio.type = 'checkbox';
-    completedRadio.name = `completed_${index}`;
-    completedRadio.checked = completed;
-    edit.textContent = 'Edit task';
+    const { index } = item;
+    const completedRadio = document.querySelector(`input[name="completed_${index}"]`);
 
```

Comment:
- [ ] Great Job making sure that both checked and unchecked items and saved to local Storage and remain on page after page reload.
<img width="944" alt="image" src="https://github.com/paulshaun-kdtk/WebPack_Linked-List/assets/52934128/3ce79f69-4654-4fc9-9995-6fcf178c897f">